src/gg.regression.unity.bots/Runtime/Scripts/RGLegacyInputUtility/RGLegacyInputWrapper.cs

```diff
@@ -185,19 +191,19 @@ private static IEnumerator InputSimLoop()
                             Vector3 newMousePosition = evt.newMousePosition;
                             if (evt.newMouseDelta.HasValue)
                             {
-                                _mousePosDelta = evt.newMouseDelta.Value;
+                                _mousePosDelta += evt.newMouseDelta.Value;
```

Comment:
this being += doesn't seem right at a glance, but I'm probably not understanding the code here

Reply:
The problem I was trying to address here is that the monkey bot might generate multiple mouse pos/scroll delta events within a single frame, and I want to have those stack up rather than only using the value of the last one. The deltas would get cleared anyways at the start of the following frame at the bottom of the event processing loop.

Though, as I write this I realize that this new behavior is probably inconsistent with how the new Input System would handle these events :/. I'll revert it back to how it was. Thanks for pointing it out!